Ecal/include/Ecal/WorkingEcalCluster.h

```diff
@@ -0,0 +1,44 @@
+/*
+   WorkingEcalCluster -- In-memory tool for working on clusters
+   */
+#ifndef ECAL_WORKINGECALCLUSTER_H_
+#define ECAL_WORKINGECALCLUSTER_H_
+
+#include <iostream>
+#include <vector>
+
+#include "Ecal/Event/EcalHit.h"
+#include "TLorentzVector.h"
+
+namespace ecal {
+
+class WorkingEcalCluster {
```

Comment:
Do we really want to call this "Working"? I'm not even sure I understand what this is supposed to mean, if to keep it maybe some more description in a comment would be good

Reply:
The "working" clusters are supposed to be the intermediate clusters that are not finished yet. As Ella pointed out, this class should be removed and replaced with using the `WorkingCluster` class which uses pointers to try to avoid as much copying of the EcalHit data.

Reply:
> The "working" clusters are supposed to be the intermediate clusters that are not finished yet. 

OK I understood this like it's working (i.e. functioning) now, as opposed to that it didnt work in the past. Should this not be better called as `TempCluster`, or `IntermediateCluster`, `InProgressCluster`, `ClusterInProgress` or something else that doesnt make it sound like this is the final product?